### PR TITLE
Adding recipe QPot

### PR DESCRIPTION
**Pull request by tdegeus**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tdegeus** [comment]
Ready for review @conda-forge/help-python-c @conda-forge/help-c-cpp

File: recipes/python-qpot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - pip
```
Comment (scopatz) [inline recipes/python-qpot/meta.yaml:23]: You probably just need pip, and can remove setuptools

File: recipes/python-qpot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  url: https://github.com/tdegeus/QPot/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (scopatz) [inline recipes/python-qpot/meta.yaml:19]: You should have make and cmake as deps here

File: recipes/python-qpot/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/python-qpot/meta.yaml:23]: nm sorry wrong file

File: recipes/qpot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - xtensor
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/QPot  # [unix]
+    - test -f ${PREFIX}/include/QPot/Static.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/QPot/QPotConfig.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\QPot\Static.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\QPotConfig.cmake (exit 0) else (exit 1)  # [win]
```
Comment (scopatz) [inline recipes/qpot/meta.yaml:31]: Can you add tests to ensure that shared libraries are being installed and not static ones?

Comment (tdegeus) [reply recipes/qpot/meta.yaml:31]: It is header only, there are no shared libraries.

File: recipes/python-qpot/meta.yaml (hunk 2 above)
Comment (tdegeus) [reply recipes/python-qpot/meta.yaml:19]: Here I use pure Python to compile. I don't think I should add make and/or cmake as additional deps

**Comment by tdegeus** [comment]
Thanks for your review @scopatz . As indicated though, I don't think changes are needed...

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**